### ISSUE TITLE
Python/sklearn Question: How to get feature names, aligning with feature_importances_

### ISSUE THREAD

**Issue by ClimbsRocks** [Microsoft/LightGBM#209]
tl;dr: Using the sklearn interface for Python, how can I access the names the model was trained on in a way that aligns with the feature_importances_?

I'm trying to take advantage of passing in a pandas DataFrame with categorical data left as `dtype='category'`. The model is training fine, and I can get the `.feature_importances_` from it. Are the feature_importances_ reported in the same order as the original DataFrame's columns? Happy to make the docs a bit clearer once I understand.

**Comment by wxchan** [comment]
It's the same order as DataFrame's columns.
